app/javascript/scss/elements/_navigation.scss

```diff
@@ -18,6 +20,8 @@
 
 .navigation__item {
   --color: #{$text-color-lightest};
+  --discord-color: #{$discord};
+  --discord-drop-shadow: drop-shadow(0 0 4px #{color.change($white, $alpha: 0.2)});
```

Comment:
You can just use rgba instead of the whole `sass:color` thing. 
```suggestion
  --discord-drop-shadow: drop-shadow(0 0 4px rgba(#{$white}, 0.2));
```

Reply:
This doesn't work because `$white` is in hex format
![demo in MDN throws an error](https://github.com/Mitcheljager/workshop.codes/assets/6181929/7d44d7f4-9eb4-4522-9a63-b33f730dcc79)

Reply:
This does seem to work, but is not supported in Firefox[1]: `rgba(from #{$white} r g b / 0.2)`

[1]: [`rgb()` Browser Compatibility](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value/rgb#browser_compatibility), Firefox doesn't support Relative RGB colors

Reply:
I know, but SCSS should transform it just fine. It's how we use it everywhere. This line for example:
https://github.com/Mitcheljager/workshop.codes/blob/6cd52cc0303345e3070ed231ed232f9eef198fd5/app/javascript/scss/elements/_dropdown.scss#L38

Perhaps it doesn't work inside of a CSS variable since it's an SCSS function. Did you try `#{rgba($white, 0.2)}`?

Reply:
That works!

Reply:
Nice!